config/sidebar.yml

```diff
@@ -400,6 +400,10 @@ articles:
       - title: "User Search"
         url: "/users/search"
 
+      - title: "User Search v3"
```

Comment:
this document should be below this one: https://auth0.com/docs/api/management/v2/user-search

![image](https://user-images.githubusercontent.com/178506/37344264-ac5197ca-26a8-11e8-821e-45d906ccba61.png)

Reply:
@siacomuzzi We want to gather all content related to users under `User Management`. https://auth0.com/docs/users We already have there content that references the Management API https://auth0.com/docs/users/search

Also once the v3 doc is ready to go public we will add versioning to it and the v2 (which will look like this, with the dropdown on the top https://auth0.com/docs/libraries/lock/v11 )

cc @erjohnson

Reply:
* `User Search` (`/users/search`) is a generic document (it's not v2 specific)
* This new document is the "v3 version" of https://auth0.com/docs/api/management/v2/user-search that's why it should be there

Reply:
@erjohnson told me the plan is to move https://auth0.com/docs/api/management/v2/user-search here, disregard my comment